docs/navigation-prop.md

```diff
@@ -108,6 +108,45 @@ React Navigation emits events to screen components that subscribe to them:
 - `didFocus` - the screen focused (if there was a transition, the transition completed)
 - `didBlur` - the screen unfocused (if there was a transition, the transition completed)
 
+
+Example:
+
+```javascript
+const didBlurSubscription = this.props.navigation.addListener('didBlur', payload => {
+  console.debug("didBlur",payload);
```

Comment:
can you format this code with prettier? you can copy and paste it into snack to do that

Reply:
done